src/main/java/dev/morling/kccli/command/GetContextCommand.java

```diff
@@ -18,17 +18,22 @@
 import javax.inject.Inject;
 
 import dev.morling.kccli.util.ConfigurationContext;
+import picocli.CommandLine;
 import picocli.CommandLine.Command;
 
 @Command(name = "get-context", description = "Get the current context")
 public class GetContextCommand implements Runnable {
 
+    @CommandLine.Parameters(index = "0", description = "Context name")
+    String contextName;
+
     @Inject
     ConfigurationContext context;
 
     @Override
     public void run() {
-        String clusterUri = context.getContext().getCluster().toASCIIString();
-        System.out.println("Current context is set to " + clusterUri);
+
+        String clusterUri = context.getContext(contextName).getCluster().toASCIIString();
```

Comment:
So this changes semantics of this command a bit, as it always has shown the currently active context. Can we keep it that way, when invoking `kcctl config get-context` without any option? And then, when calling `kcctl config get-context --name <NAME>`, show the specified one? I.e. make the name an optional parameter.

Reply:
sure, that makes sense, I even did but could not get how to get the optional params, but your suggestion will help now to do it